frontend/test/src/reducer/workspaceList.spec.js

```diff
@@ -91,18 +91,18 @@ describe('workspaceList reducer', () => {
     })
 
     describe(`${ADD}/${WORKSPACE_MEMBER}`, () => {
-      const rez = workspaceList(
-        [
-          ...initialState,
-          {
-            ...serializedFirstWorkspaceFromApi,
-            memberList: []
-          }
-        ],
-        addWorkspaceMember(globalManagerFromApi, firstWorkspaceFromApi.workspace_id, { ...globalManagerFromApi, do_notify: true, role: ROLE.workspaceManager })
-      )
-
       it('should return a workspace list with the member correctly added in the right workspace', () => {
+        const rez = workspaceList(
+          [
+            ...initialState,
+            {
+              ...serializedFirstWorkspaceFromApi,
+              memberList: []
+            }
+          ],
+          addWorkspaceMember(globalManagerFromApi, firstWorkspaceFromApi.workspace_id, { ...globalManagerFromApi, do_notify: true, role: ROLE.workspaceManager })
```

Comment:
Maybe write this call or this object on several lines (but feel free not to)

Reply:
Fixed